profiler/test/Datadog.Profiler.IntegrationTests/SmokeTests/CpuLimitTest.cs

```diff
@@ -0,0 +1,45 @@
+// <copyright file="CpuLimitTest.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Profiler.SmokeTests;
+using FluentAssertions;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Profiler.IntegrationTests.SmokeTests
+{
+    public class CpuLimitTest
+    {
+        private readonly ITestOutputHelper _output;
+
+        public CpuLimitTest(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+        [Trait("Category", "CpuLimitTest")]
```

Comment:
Shouldn't we name the category `ContainerTest` instead ?
in the future we would have Container-based tests...

Reply:
I thought about it, but if we have container test we probably don't want to launch them multiple times with different CPU settings